temporalcli/commandsgen/commands.yml

```diff
@@ -374,6 +375,99 @@ commands:
     option-sets:
       - workflow reference
 
+  - name: temporal activity pause
+    summary: Pause an Activity
+    description: |
+      Pausing an activity means:
+        - If the activity is currently waiting for a retry or is running 
+      and subsequently fails, it will not be rescheduled until it is unpaused.
+      - If the activity is paused, calling this method will have no effect.
+      - If the activity is running and finishes successfully, 
+        the activity will be completed.
+      - If the activity is running and finishes with failure:
+        * if there is no retry left - the activity will be completed.
+        * if there are more retries left - the activity will be paused.
```

Comment:
This formatting is a bit strange. Some `-` are indented, some aren't, some parts are "If ..., ..." and some are "if ... - ...", etc.